vignettes/relase-process.qmd

```diff
@@ -0,0 +1,198 @@
+---
+title: "The Release Process"
+description: >
+  Reference for how to perform R package releases in the Hubverse
+vignette: >
+  %\VignetteIndexEntry{The Release Process}
+  %\VignetteEngine{quarto::html}
+  %\VignetteEncoding{UTF-8}
+knitr:
+  opts_chunk: 
+    collapse: true
+    comment: '#>'
+---
+
+This document is adapted for The Hubverse from [The Carpentries Hubverse Developer's Handbook](https://carpentries.github.io/workbench-dev/releases.html) (c) The Carpentries. 
+
+## Checklist
+
+ - [ ] Create a new branch from `main` called `release-X.Y.Z`
+ - [ ] Update version number in DESCRIPTION and check that the `Remotes:` is
+       up-to-date
+ - [ ] Update NEWS for the changes in this version
+ - [ ] commit, push changes, and create pull request
+ - [ ] get a review from another member of the Hubverse dev team and merge on approval
+ - [ ] add new tag (signed preferred) with the name "<package> X.Y.Z" and push
+
+      ```bash
+      # example: create a signed (-s) tag for sandpaper 3.3.3
+      git tag -s 3.3.3 -m '<short explanation of what changed>'
+      git push --tags
+      ```
+ - [ ] create a release on github from the new tag
+
+
+## Background
+
+Each of these packages are available on the [Hubverse R-Universe][r-universe]
+and new versions are checked for hourly. This allows folks to get up-to-date
+versions of The Hubverse packages built for their system without running out
+of GitHub API query attempts.
+
+In order to maintain quality, packages are only sent to the R-Universe if they
+have been formally released on GitHub ([as specified in the packages.json
+configuration
+file](https://github.com/hubverse-org/hubverse-org.r-universe.dev/blob/HEAD/packages.json#L16-L33)).
+This allows us to incrementally add new experimental features without changing
+the stable deployments.
+
+## Versioning
+
+The hubverse is built using very basic semantic versioning using the
+`X.Y.Z[.9000]` pattern. 
+
+[`X`]{#major}
```

Comment:
While the recap in semantic version is super useful, it might be better to use hubverse examples as points of reference

Reply:
Oh good point! I think the best example might be the `schemas` repo since that has major version changes (albeit no minor version changes, but the line between minor and patch is often subjective)

Reply:
Aaand I realized that I had forgotten to scrub mentions of `$OLDJOB` packages, which now puts your comment in even more urgent context 🙈